src/reducer.js

```diff
@@ -70,7 +120,10 @@ export default (state = initialState, action = {}) => {
         authError: null,
         profile: null,
         isLoading: false,
-        data: {}
+        data: {},
+        timestamp: {},
```

Comment:
yes sure, i missed that. can you add also the others ? (isRequesting: {}, isRequested: {}).